zio/csvio/ztests/extraneous-quotes.yaml

```diff
@@ -0,0 +1,14 @@
+script: zq -i csv -z '*' -
```

Comment:
Make this a Zed test instead of a shell test.

Reply:
You can't specify input flags for zed tests.

Reply:
Not a problem: ztest uses `anyio.NewReader`, which will figure out that the input is CSV if this is working.

Reply:
This is the error I get:
```
/Users/nibs/Code/zed/zio/csvio/ztests/extraneous-quotes.yaml: format detection error
                	tzng: line 1: bad format
                	zeek: line 1: bad types/fields definition in zeek header
                	zjson: line 1: invalid character 'i' in literal false (expecting 'a')
                	zson: identifier "field1" must be enum and requires decorator
                	zng: zng type ID out of range
                	csv: auto-detection not supported
                	json: auto-detection not supported
                	parquet: auto-detection not supported
                	zst: auto-detection not supported
```

Reply:
I guess I'm a dummy. Sorry about that!